pkg/job/discovery_test.go

```diff
@@ -425,9 +421,6 @@ func Test_getFilteredMetricDatas(t *testing.T) {
 				if *got.AddCloudwatchTimestamp != *tt.wantGetMetricsData[i].AddCloudwatchTimestamp {
 					t.Errorf("getFilteredMetricDatas().AddCloudwatchTimestamp = %v, want %v", *got.AddCloudwatchTimestamp, *tt.wantGetMetricsData[i].AddCloudwatchTimestamp)
 				}
-				if *got.AddHistoricalMetrics != *tt.wantGetMetricsData[i].AddHistoricalMetrics {
```

Comment:
why do we remove this test check again?

Reply:
`AddHistoricalMetrics` is not in the model anymore but is pass as a var

Reply:
but it's still present in the model here? https://github.com/grid-x/yet-another-cloudwatch-exporter/blob/d7147db6a6ad9349fb75867b29041b00dba20765/pkg/model/model.go#L56
I'm not too deep into the code - it just seems weird to me to add this field explicitly to all the expected test data, but then not actually check whether the field matches the expected value. If it doesn't matter, then we should remove initializing it e.g. here https://github.com/grid-x/yet-another-cloudwatch-exporter/blob/d7147db6a6ad9349fb75867b29041b00dba20765/pkg/job/discovery_test.go#L93; if it does matter and we expect it to be `false`, we should re-add this line here

Reply:
true, in the PR `AddHistoricalMetrics` was a `bool` not a `*bool`. So it was false by default, now it's passed throw the functions @guelfey